### PR TITLE
[Static pages] Add Caregivers content toggle

### PR DESCRIPTION
**Pull request by ncksllvn**
## Description
This PR creates a feature toggle on `/health-care/family-caregiver-benefits/comprehensive-assistance/` so that Flipper can be used to toggle the user flow into the upcoming Caregivers online form.

https://github.com/department-of-veterans-affairs/va.gov-team/issues/8853
## Testing done
- Added `<div data-widget-type="caregiver-content-toggle"></div>` into a vagov-content page
- Went to that page on localhost
- Turned on API and toggled the flag at `api/flipper`
- Observed on/off states

### Off state
image

### On state
image

## Acceptance criteria
- [ ] Caregiver CTA can be toggled

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/platform/utilities/feature-toggles/featureFlagNames.js [hunk 1]
```diff
@@ -26,4 +26,5 @@ export default Object.freeze({
   gibctEstimateYourBenefits: 'gibctEstimateYourBenefits',
   form526OriginalClaims: 'form526OriginalClaims',
   vaViewDependentsAccess: 'vaViewDependentsAccess',
+  allowOnline1010cgSubmissions: 'allowOnline1010cgSubmissions',
```
Comment (ncksllvn) [inline src/platform/utilities/feature-toggles/featureFlagNames.js:29]: The API already had this flag registered for this use case, https://github.com/department-of-veterans-affairs/vets-api/pull/4220. It just hadn't been registered in the frontend.

**Review by kelsonic — APPROVED** [review]
LGTM! Nicely done! 🎉Where did you get the content/design for this PR btw?

**Comment by ncksllvn** [comment]
@kelsonic Thanks! There were images in this screenshot - https://github.com/department-of-veterans-affairs/va.gov-team/issues/8853#issuecomment-628270886. I also just copied the content out of `/health-care/family-caregiver-benefits/comprehensive-assistance/` to create the off state.

**Comment by ncksllvn** [comment]
Hi @department-of-veterans-affairs/vsa-caregiver-frontend - this is the work for creating a feature toggle on `/health-care/family-caregiver-benefits/comprehensive-assistance/` to control the user flow to your form.

**Review by U-DON — APPROVED** [review]

**Pull request merged**